kolibri/core/assets/src/vue/content-renderer/download-button.vue

```diff
@@ -6,15 +6,15 @@
       class="dropdown-button"
       @click="toggleDropdown"
       aria-haspopup="true">
-      <span class="dropdown-button-text" :class="{uparrow: dropdownopen}">
+      <span class="dropdown-button-text" :class="{'dropdown-button-text-open': dropdownOpen}">
         {{ $tr('downloadContent') }}
       </span>
     </button>
     <ul
       ref="dropdownitems"
       class="dropdown-items"
-      :class="{ dropdownopen: dropdownopen }"
-      :aria-hidden="dropDownOpenText"
+      :class="{ 'dropdown-open': dropdownOpen }"
+      :aria-hidden="dropdownOpenText"
```

Comment:
Seems to work for IE11 and up (https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden).

I can't quite tell from the second link whether `hidden` will translate to `aria-hidden` in browser we support, but it seems that way?